howtos/custom_generators.rst

```diff
@@ -199,83 +121,85 @@ reference in conanfile.txt.
     [generators]
     Premake
 
-**main.cpp**
-
 .. code-block:: cpp
+   :caption: *main.cpp*
 
     #include "hello.h"
     
-    int main (void){
+    int main (void) {
         hello();
     }
-    
-**premake4.lua**
 
 .. code-block:: lua
+   :caption: *premake4.lua*
+
+    -- premake4.lua
 
-    #!lua
-    
     require 'conanpremake'
-    
-    -- A solution contains projects, and defines the available configurations
-    solution "MyApplication"
-       configurations { "Debug", "Release" }
-       includedirs { conan_includedirs }
-       libdirs { conan_libdirs }
-       links { conan_libs }
-       -- A project defines one build target
-       project "MyApplication"
-          kind "ConsoleApp"
-          language "C++"
-          files { "**.h", "**.cpp" }
-     
-          configuration "Debug"
-             defines { "DEBUG" }
-             flags { "Symbols" }
-    
-          configuration "Release"
-             defines { "NDEBUG" }
-             flags { "Optimize" }
 
+    -- A solution contains projects, and defines the available configurations solution "MyApplication"
+
+    configurations { "Debug", "Release" }
+    includedirs { conan_includedirs }
+    libdirs { conan_libdirs }
+    links { conan_libs }
+
+    -- A project defines one build target
+
+    project "MyApplication"
+        kind "ConsoleApp"
+        language "C++"
+        files { "**.h", "**.cpp" }
+
+        configuration "Debug"
+            defines { "DEBUG" }
+            flags { "Symbols" }
+
+        configuration "Release"
+            defines { "NDEBUG" }
+            flags { "Optimize" }
 
 Let's install the requirements and build the project:
 
 .. code-block:: bash
 
-   $ conan install . -s compiler=gcc -s compiler.version=4.9 -s compiler.libcxx=libstdc++ --build
-   $ premake4 gmake
-   $ make (or mingw32-make if in windows-mingw)
-   $ ./MyApplication
-   Hello World!
+    $ conan install . -s compiler=gcc -s compiler.version=4.9 -s compiler.libcxx=libstdc++ --build
+    $ premake4 gmake
+    $ make (or mingw32-make if in windows-mingw)
+    $ ./MyApplication
+    Hello World!
```

Comment:
I need to read here something like  _"after calling `conan install ...` there is anew file, `conanpremake.lua` (where the name comes from) that is created with...."_.